style.css

```diff
@@ -0,0 +1,3 @@
+.header {
+  color: bisque;
+}
```

Comment:
- [ ] **In the README.md** file: I think it would be better if you add a good description in your README telling in detail what about your project to let users know the content and make it professional.